CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-meter ChangeLog
 
+## 5.2.1 - 2023-08-xx
+
+### Fixed
+- Change `return` to `continue` when getting a duplicate error adding
```

Comment:
You have described the technical solution here, but it would be good to describe the symptom that the end-user might experience.  In addition to what you have...

"Prior to this fix, some of the mock meters may not be properly created."